beacon-chain/casper/validator.go

```diff
@@ -178,9 +178,9 @@ func TotalActiveValidatorDeposit(dynasty uint64, validators []*pb.ValidatorRecor
 }
 
 // TotalActiveValidatorDepositInEth returns the total deposited amount in ETH for all active validators.
-func TotalActiveValidatorDepositInEth(dynasty uint64, validators []*pb.ValidatorRecord) uint64 {
-	totalDeposit := TotalActiveValidatorDeposit(dynasty, validators)
-	depositInEth := totalDeposit / uint64(params.GetConfig().EtherDenomination)
+func TotalActiveValidatorDepositInEth(validators []*pb.ValidatorRecord) uint64 {
+	totalDeposit := TotalActiveValidatorDeposit(validators)
+	depositInEth := totalDeposit / uint64(params.GetConfig().GWEIPerETH)
```

Comment:
can you change params name to just `Gwei` or `GweiDenomination` , the name Gwei is self explanatory we dont need the `perETH`